src/simularium/VisData.ts

```diff
@@ -23,6 +23,7 @@ const EOF_PHRASE: Uint8Array = new TextEncoder().encode(
 class VisData {
     private frameCache: AgentData[][];
     private frameDataCache: FrameData[];
+    private disableCache: boolean;
```

Comment:
as others have said, this should be the affirmative version of the variable, like `useCache` here.

Reply:
Ok, I think I'll change it back then, since there seems to be a majority preferring the affirmative.